ddtrace/contrib/django/utils.py

```diff
@@ -139,7 +139,6 @@ def get_request_uri(request):
 
 
 def _before_request_tags(pin, span, request):
-    trace_utils.activate_distributed_headers(pin.tracer, int_config=config.django, request_headers=request.META)
```

Comment:
Just curious, why are we removing this?

Reply:
Hmm it was actually redundant here since we call it before this function is called in `traced_get_response`. I think it might have been an error introduced in #2657. I might just pull this change out separately since it's not really related to this PR. :+1:

Reply:
Fixed in https://github.com/DataDog/dd-trace-py/pull/2716